agent/acs/handler/attach_eni_handler_test.go

```diff
@@ -28,9 +28,9 @@ import (
 )
 
 const (
-	eniMessageId = "123"
-	randomMAC    = "00:0a:95:9d:68:16"
-	waitTimeout  = 10
+	eniMessageId      = "123"
+	randomMAC         = "00:0a:95:9d:68:16"
+	waitTimeoutMillis = 10
```

Comment:
If it were `waitTimeout`, it would need to be equal to `10 * time.Millisecond` rather than its current value.  I don't see any reason to change the name.